build.maven/iar/pom.xml

```diff
@@ -14,7 +14,7 @@
 
 	<properties>
 		<antrun.version>1.7</antrun.version>
-		<project.build.plugin.version>8.0.4</project.build.plugin.version>
+		<project.build.plugin.version>8.0.5-SNAPSHOT</project.build.plugin.version>

```

Comment:
this will cause issues on the customer side... when they re-import the project (since this snapshot repository is not visible in the pom)
... please adapt a cool fix for this from master